packages/web-app/src/routes/index.js

```diff
@@ -27,6 +28,6 @@ router.use(`/${views.invalidAppealDetails}`, handleAppealAlreadyReviewed, invali
 router.use(`/${views.missingOrWrong}`, handleAppealAlreadyReviewed, missingOrWrongAppealDetails);
 router.use(`/${views.document}`, documentsServiceProxy);
 router.use(`/${views.appealAlreadyReviewed}`, appealAlreadyReviewed);
-router.use(`/${views.checkAndConfirm}`, checkAndConfirmDetails);
+router.use(`/${views.questionnairesForReview}/${views.checkAndConfirm}`, checkAndConfirm);
```

Comment:
It looks like this is using the Appeals Check and Confirm router rather than the new Questionnaire one so as it is it'll be pointing to the wrong page?

We also want to keep the Appeals Check and Confirm page so the Questionnaire one should be added rather than replacing it.

Reply:
yeah this was a mistake, will add back.